package/MDAnalysis/core/topologyattrs.py

```diff
@@ -2032,6 +2045,239 @@ def total_charge(group, compound='group'):
     transplants[GroupBase].append(
         ('total_charge', total_charge))
 
+    @warn_if_not_unique
+    @_pbc_to_wrap
+    @check_wrap_and_unwrap
+    def dipole_moment(group, wrap=False, unwrap=False, 
+        compound='group', center="mass"):
+        r"""Dipole vector of the group.
+
+        Computes the dipole moment of :class:`Atoms<Atom>` in the group.
```

Comment:
Would be great if you could put the mathematical definition as you did for quadrupole below :)

Reply:
Clarify how the center is chosen (`center`) and remind the user that it makes no difference for neutral groups of atoms but for charged ones, the results depend on it.

Perhaps add a test for neutral molecules and charged ones that confirm center independence/dependence.